website/docs/contributing/writing-guide.md

```diff
@@ -43,26 +43,24 @@ Use sentence case for headers: `### Some header with URL`, not `### Some Header
 
 ## Links
 
-Please use markdown file paths for links, not URL paths,
-because it works for both editors/IDEs (Ctrl/⌘+click works) and Docusaurus.
-Always add `.md` extension to the file paths.
-Use relative paths for links to files in the same directory, in a sub-directory, or in a parent directory.
+Please use relative `.md` file paths for links.
+It is required for [documentation versioning](https://docusaurus.io/docs/versioning#link-docs-by-file-paths).
 
 Examples:
 
 To link to a file in the same directory, use the file name.
 
 - `[file in the same directory](writing-guide.md)`
 
-To link to a file in a parent directory, prefix with `../` to go up one directory level.
+To link to file in a different directory, specify the relative file path.
 
-- `[file in a parent directory](../telemetry.md)`
+- `[file in a different directory](../basic-operations/read.md)`.
 
-To link to file in a subdirectory, specify the file path along with its respective directory or directories, such as: `subdirectory/file.md`.
+# FIXME `/category/` does not work correctly
 
-To link to a directory or category, prefix the directory name with `/category/`.
+To link to category, prefix the directory name with `/category/`.
 
-- `[configuration directory](/category/configuration/)`
+- [configuration directory](/category/configuration/)
```

Comment:
@Fashander `/category/xxx` leads to the latest version, not to the same version

Reply:
In the docs, it suggests we link to file paths with `.md` extensions. 

https://docusaurus.io/docs/versioning#link-docs-by-file-paths

Also, there's an open issue in Docusaurus that mentions it (https://github.com/facebook/docusaurus/issues/6041). We can choose not to include it, since it only appears once within the docs (writing-guide).